rdiodj/static/css/style.less

```diff
@@ -573,6 +525,7 @@ img {
 
 .presence {
 	margin-left: -15px;
+	margin-bottom: 0.4em;
```

Comment:
In general, I think relative sizes (% or em) are better. The `margin-left: -15px;` was a quick fix to align image profile.